x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -272,9 +272,9 @@ public enum Cap {
         MV_PSERIES_WEIGHTED_SUM,
 
         /**
-         * Support for match operator
+         * Support for match operator as a colon. Previous support for match operator as MATCH has been removed
          */
-        MATCH_OPERATOR(true),
+        MATCH_OPERATOR_COLON(true),
```

Comment:
I've removed the previous MATCH_OPERATOR to ensure BwC tests are green. Otherwise, 8.16 BwC tests would try to use the previous syntax for the MATCH operator.

Removing the previous operator and changing it makes everything easier as we're not supporting the previous capability anymore, and introducing a new one.

Reply:
Thanks for the catch! Done in 71328d6558f255ac21a98626a2f0a488a54631e6